src/main/java/Storage.java

```diff
@@ -0,0 +1,122 @@
+/** This file deals with all file reading from and writing to. */
+
+import types.Deadlines;
+import types.Party;
+import types.Task;
+import types.Todo;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public class Storage {
+    public static void addToList(Path path, String type, String desc) {
+        String line = type + "," + 0 + "," + desc + "\n";
+        try {
+            Files.write(path, line.getBytes(), StandardOpenOption.APPEND);
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+    }
+
+    public static void addToList(Path path, String type, String desc, LocalDate deadline) {
+        String line = type + "," + 0 + "," + desc + "," + deadline + "\n";
+        try {
+            Files.write(path, line.getBytes(), StandardOpenOption.APPEND);
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+    }
+
+    public static void addToList(Path path, String type, String desc, LocalDate from, LocalDate to) {
+        String line = type + "," + 0 + "," + desc + "," + from + "," + to + "\n";
+        try {
+            Files.write(path, line.getBytes(), StandardOpenOption.APPEND);
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+    }
+    public static ArrayList<Task> getLastList() {
+        ArrayList<Task> finalList = new ArrayList<>();
+
+        try {
+            Path path = Paths.get("barbie.txt");
+
+            if (!Files.exists(path)) {
+                Files.createFile(path);
+                System.out.println("[A new list created for current user]");
+            } else {
+                System.out.println("-------------------------------------------------");
+                System.out.println("[A current list is being used for current user]");
+                Files.readAllLines(path).forEach(x -> {
+
+                    String[] taskParts = x.split(",");
+                    String taskType = taskParts[0];
+                    String taskStatus = taskParts[1];
+                    String desc = taskParts[2];
+                    Task task;
+
+                    if (Objects.equals(taskType, "T")) {
+                        task = new Todo(desc);
+                    } else if (Objects.equals(taskType, "D")) {
+                        task = new Deadlines(desc, LocalDate.parse(taskParts[3]));
+                    } else if (Objects.equals(taskType, "P")) {
+                        task = new Party(desc, LocalDate.parse(taskParts[3]), LocalDate.parse(taskParts[4]));
```

Comment:
```suggestion
                    if (String.equals(taskType, "T")) {
                        task = new Todo(desc);
                    } else if (String.equals(taskType, "D")) {
                        task = new Deadlines(desc, LocalDate.parse(taskParts[3]));
                    } else if (String.equals(taskType, "P")) {
                        task = new Party(desc, LocalDate.parse(taskParts[3]), LocalDate.parse(taskParts[4]));
```
Consider using string.equals for this scenario as it is more specific to the data types.

Reply:
I didn't notice this, thanks!!